cron/src/main/scala/com/criteo/cuttle/cron/CronModel.scala

```diff
@@ -23,6 +24,7 @@ private[cron] case class ScheduledAt(instant: Instant, delay: FiniteDuration)
 private[cron] case class CronState(logger: Logger) {
   private val executions = Ref(Map.empty[CronJob, Either[Instant, CronExecution]])
   private val paused = Ref(Map.empty[CronJob, PausedJob])
+  private val deferreds = Ref(Map.empty[CronJob, Deferred[IO, Unit]])(optManifest)
```

Comment:
I suggest to use Deferred[IO, ScheduledAt]. It will be cleaner.

Reply:
Done